shared/templates/service_disabled/ansible.template

```diff
@@ -3,27 +3,30 @@
 # strategy = disable
 # complexity = low
 # disruption = low
+
 {{%- if init_system == "systemd" %}}
-- name: Block Disable service {{{ SERVICENAME }}}
-  block:
-    - name: Disable service {{{ SERVICENAME }}}
-      block:
-        - name: Disable service {{{ SERVICENAME }}}
-          systemd:
-            name: "{{{ DAEMONNAME }}}.service"
-            enabled: "no"
-            state: "stopped"
-            masked: "yes"
-      rescue:
-        - name: "Intentionally ignored previous 'Disable service {{{ SERVICENAME }}}' failure, service was already disabled"
-          meta: noop
+
+- name: "{{{ rule_title }}} - Collect systemd Services Present in the System"
+  command: systemctl -q list-unit-files --type service
```

Comment:
Please, use FQCN when calling the Ansible modules:
https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#ansible

It would be great to update all existing tasks to also use FQCN.